<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartDesignGui::TaskThicknessParameters</class>
 <widget class="QWidget" name="PartDesignGui::TaskThicknessParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="message">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="buttonRefAdd">
       <property name="toolTip">
        <string>In toggle mode, you can select geometry either
in the list below, or in the 3D view to toggle
its state.

In normal mode, you can only add new geometry
by selection. Existing geometry references can
be removed using the Delete key.</string>
       </property>
       <property name="text">
        <string>Face selection toggle mode</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidgetReferences">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Thickness</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Gui::QuantitySpinBox" name="Value" native="true">
       <property name="keyboardTracking" stdset="0">
        <bool>false</bool>
       </property>
       <property name="unit" stdset="0">
        <string notr="true">mm</string>
       </property>
       <property name="minimum" stdset="0">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum" stdset="0">
        <double>999999999.000000000000000</double>
       </property>
       <property name="singleStep" stdset="0">
        <double>0.100000000000000</double>
       </property>
       <property name="value" stdset="0">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="modeComboBox">
       <item>
        <property name="text">
         <string>Skin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pipe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recto Verso</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Join Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="joinComboBox">
       <item>
        <property name="text">
         <string>Arc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Intersection</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkMakeOffset">
     <property name="text">
      <string>Make offset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkIntersection">
     <property name="text">
      <string>Intersection</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkReverse">
     <property name="text">
      <string>Make thickness inwards</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonRefAdd</tabstop>
  <tabstop>treeWidgetReferences</tabstop>
  <tabstop>Value</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>joinComboBox</tabstop>
  <tabstop>checkMakeOffset</tabstop>
  <tabstop>checkIntersection</tabstop>
  <tabstop>checkReverse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
